src/Propel/Generator/Util/BehaviorLocator.php

```diff
@@ -204,14 +204,17 @@ private function getCoreBehavior($name)
      *
      * @return array
      */
-    private function loadBehaviors($composerLock)
+    private function loadBehaviorsFromLockFile($composerLock)
```

Comment:
Do we need to switch the method name here?
The argument is already clear to be the lock file.

Reply:
When you use the method, you get `$this->loadBehaviors($anyNameYouDareToUse)`. So that's a problem. Not with `$this->loadBehaviorsFromLockFile($doesAnyoneElseSmellToast)`.
 Also, the method does not load behaviors generally, so why mislead?

Reply:
Can you please add the docblock type for the argument?

Reply:
What do you mean? Docblock has parameters with types and even a description (not to my credit, that was already there)

Reply:
And if you modify a private method, you can also add types for argument and return where applicable.
As there is no BC issue in that case.

Reply:
sure, no problem, I have added type hints